src/components/Trade/TradeInput.tsx

```diff
@@ -180,6 +214,9 @@ export const TradeInput = ({ history }: RouterProps) => {
                   onChange(e.value)
                   if (e.value !== value) {
                     const action = !!e.value ? TradeActions.FIAT : undefined
+
+                    valueTrade = value?.length
+
```

Comment:
What does this do?

Reply:
the variable "value" can only be accessible in that function scope, that makes it globally accessible , and i want the length, the "?" is if it returns null